common/src/main/java/uk/co/harieo/forceop/common/DataConverter.java

```diff
@@ -0,0 +1,70 @@
+package uk.co.harieo.forceop.common;
+
+public class DataConverter {
+
+	private static final char[] HEX_ARRAY = "0123456789ABCDEF".toCharArray();
+
+	/**
+	 * Converts an array of bytes into hex in string format
+	 *
+	 * @param bytes to convert to hex
+	 * @return the hex encoded string
+	 */
+	public static String convertBinaryToHex(byte[] bytes) {
```

Comment:
```suggestion
	public static String convertBytesToHex(byte[] bytes) {
```

I think this suits it a bit better, I wasn't sure if this was talking about a binary string as bytes or if it was converting the bytes into hexadecimal